src/vs/workbench/api/test/browser/extHostApiCommands.test.ts

```diff
@@ -1486,26 +1490,6 @@ suite('ExtHostLanguageFeatureCommands', function () {
 		assert.strictEqual(outgoing[0].name, 'SUB');
 	});
 
-	test('selectionRangeProvider on inner array always returns outer array #91852', async function () {
-
-		disposables.push(extHost.registerSelectionRangeProvider(nullExtensionDescription, defaultSelector, <vscode.SelectionRangeProvider>{
-			provideSelectionRanges(_doc, positions) {
-				const [first] = positions;
-				return [
-					new types.SelectionRange(new types.Range(first.line, first.character, first.line, first.character)),
-				];
-			}
-		}));
-
-		await rpcProtocol.sync();
-		const value = await commands.executeCommand<vscode.SelectionRange[]>('vscode.executeSelectionRangeProvider', model.uri, [new types.Position(0, 10)]);
-		assert.strictEqual(value.length, 1);
-		assert.strictEqual(value[0].range.start.line, 0);
-		assert.strictEqual(value[0].range.start.character, 10);
-		assert.strictEqual(value[0].range.end.line, 0);
-		assert.strictEqual(value[0].range.end.character, 10);
-	});
-
 	test('selectionRangeProvider on inner array always returns outer array #91852', async function () {
 
```

Comment:
This is just a name duplication but a (slighty) different test (provider returns one or two elements) so we should keep both and simply rename them